retriever/engines/postgres.py

```diff
@@ -17,21 +18,27 @@ class engine(Engine):
         "bool": "boolean",
     }
     max_int = 2147483647
+    pgpass_file = io.open(PGPASS_FILE_PATH, 'r')
+    pgpass_credentials = pgpass_file.split(':') # Split the credentials file of the format hostname:port:database:username:password
```

Comment:
line to long,
```

# Split the credentials file of the format
# hostname:port:database:username:password
pgpass_credentials = pgpass_file.split(':')

```
Some thing like this would be fine